app/assets/javascripts/chf_ga_events.js

```diff
@@ -2,13 +2,15 @@
 // https://github.com/samvera/hyrax/blob/1e504c200fd9c39120f514ac33cd42cd843de9fa/app/assets/javascripts/hyrax/ga_events.js
 //
 // We add our own more flexible.
+// Can be used by a container or directly by a target; just requires the 'ga-event-data' class on the element holding the data.
 //
 // https://developers.google.com/analytics/devguides/collection/gajs/eventTrackerGuide
 
-$(document).on('click', '*[data-analytics-event]', function(e) {
+$(document).on('click', '.ga-event-data', function(e) {
   _gaq.push(['_trackEvent',
```

Comment:
if you're okay with this choice, I can change the name of the class if you like. `.log-analytics-event`?